pkg/trie/triedb/action.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2024 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package triedb
+
+type Action interface {
```

Comment:
I want to make sure I am understanding this right: This `Action` and the associated types below are just wrappers around nodes, and when they are used how they are handled is based on the type, is this correct?

Maybe a short comment here could be helpful

Reply:
Yes, this structure is used in the `Triedb.inspect` method to apply changes in our in-memory structure that hold the temporal trie changes.
I'll add a comment

Reply:
Good point, I'll change it